src/app/gui/map/mapservice.js

```diff
@@ -197,13 +197,9 @@ class OlMapViewer {
         { duration: 300, center: coordinates },
         (zoom ? ({ zoom, duration: 300 }) : ({ duration: 300, resolution: view.getResolution() })
       ));
-    } else {
-      view.setCenter(coordinates);
-    }
+    } else { view.setCenter(coordinates) }
 
-    if (zoom && !animate) {
-      view.setZoom(zoom);
-    }
```

Comment:
@volterra79 this is bad practice.

Inline conditions should only be used when they can help you improve overall code readability/verbosity, eg. when you have "many short" `if` statements:

```js
// TOT: 10 conditions → 10 lines

if (condition_1)  { command_1(); }
if (condition_2)  { command_2(); }
if (condition_3)  { command_3(); }
if (condition_4)  { command_4(); }
if (condition_5)  { command_5(); }
if (condition_6)  { command_6(); }
if (condition_7)  { command_7(); }
if (condition_8)  { command_8(); }
if (condition_9)  { command_9(); }
if (condition_10) { command_10(); }
```

```js
// TOT: 10 conditions → 40 lines

if (condition_1) {
  command_1();
}

if (condition_2) {
 command_2();
}

if (condition_3) {
  command_3();
}

if (condition_4) {
  command_4();
}

if (condition_5) {
  command_5();
}

if (condition_6) {
  command_6();
}

if (condition_7) {
  command_7();
}

if (condition_8) {
  command_8();
}

if (condition_9) {
  command_9();
}

if (condition_10) {
  command_10();
}
```